diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64b979a..3f785d3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -359,9 +359,9 @@ if(NOT BROTLI_BUNDLED_MODE)
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 endif()  # BROTLI_BUNDLED_MODE
 
-INSTALL(FILES "docs/brotli.1" DESTINATION "${SHARE_INSTALL_PREFIX}/man/man1")
+INSTALL(FILES "docs/brotli.1" DESTINATION "${CMAKE_INSTALL_PREFIX}/share/man/man1")
 INSTALL(FILES docs/constants.h.3 docs/decode.h.3 docs/encode.h.3 docs/types.h.3
-  DESTINATION "${SHARE_INSTALL_PREFIX}/man/man3")
+  DESTINATION "${CMAKE_INSTALL_PREFIX}/share/man/man3")
 
 if (ENABLE_COVERAGE STREQUAL "yes")
   SETUP_TARGET_FOR_COVERAGE(coverage test coverage)
